src/ape/managers/converters.py

```diff
@@ -170,6 +192,7 @@ def _converters(self) -> Dict[Type, List[ConverterAPI]]:
             AddressType: [address_api_converter, hex_address_converter],
             bytes: [hex_converter],
             int: [],
+            str: [],
```

Comment:
We index here by the output type, not input type. Also, we want to add our new converter here
```suggestion
            int: [timestamp_converter],
```